Formula/apache-pulsar.rb

```diff
@@ -0,0 +1,70 @@
+class ApachePulsar < Formula
+  desc "Cloud-native distributed messaging and streaming platform"
+  homepage "https://pulsar.apache.org/"
+  url "https://www.apache.org/dyn/mirrors/mirrors.cgi?action=download&filename=pulsar/pulsar-2.8.0/apache-pulsar-2.8.0-src.tar.gz"
+  mirror "https://archive.apache.org/dist/pulsar/pulsar-2.8.0/apache-pulsar-2.8.0-src.tar.gz"
+  sha256 "0e161a81c62c7234c1e0c243bb6fe30046ec1cd01472618573ecdc2a73b1163b"
+  license "Apache-2.0"
+  head "https://github.com/apache/pulsar.git", branch: "master"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "cppunit" => :build
+  depends_on "libtool" => :build
+  depends_on "maven" => :build
+  depends_on "pkg-config" => :build
+  depends_on "protobuf" => :build
+  depends_on arch: :x86_64
+  depends_on "openjdk@11"
+
+  def install
+    # Missing executable permission reported upstream: https://github.com/apache/pulsar/issues/11833
+    chmod "+x", "src/rename-netty-native-libs.sh"
+    with_env("TMPDIR" => buildpath, **Language::Java.java_home_env("11")) do
+      system("mvn", "-X", "clean", "package", "-DskipTests", "-Pcore-modules")
+    end
+    built_version = if build.head?
+      # This script does not need any particular version of py3 nor any libs, so both
+      # brew-installed python and system python will work.
+      Utils.safe_popen_read("python3", "src/get-project-version.py").strip
+    else
+      version
+    end
+    binpfx = "apache-pulsar-#{built_version}"
+    system "tar", "-xf", "distribution/server/target/#{binpfx}-bin.tar.gz"
+    libexec.install "#{binpfx}/bin", "#{binpfx}/lib", "#{binpfx}/instances", "#{binpfx}/conf"
+    (libexec/"lib/presto/bin/procname/Linux-ppc64le").rmtree
+    share.install "#{binpfx}/examples"
+    share.install "#{binpfx}/licenses"
+    (var/"log/pulsar").mkpath
+    (etc/"pulsar").mkpath
+    (etc/"pulsar").install_symlink libexec/"conf"
+
+    Pathname.glob("#{libexec}/bin/*") do |path|
+      if !path.fnmatch?("*common.sh") && !path.directory?
+        bin_name = path.basename
+        (bin/bin_name).write_env_script libexec/"bin/#{bin_name}", Language::Java.java_home_env("11")
```

Comment:
```suggestion
        (bin/bin_name).write_env_script libexec/"bin"/bin_name, Language::Java.java_home_env("11")
```
We don't need interpolation here.